apps/web/__tests__/support/pageObjects/CheckoutPageObject.ts

```diff
@@ -2,6 +2,9 @@ import type { AddressFixtureOverride } from '~/__tests__/types';
 import { PageObject } from './PageObject';
 
 export class CheckoutPageObject extends PageObject {
+  shouldShowPreferredDeliverySurcharge() {
```

Comment:
we should start to create new PageObjects once we add more features. For example mollie should not add its helpers into this page object. We should create a MolliePageObject or DHLPageObject. Change if it you want, up to you 👍